### DISCUSSION TITLE
format of sound  data (mono/stereo, 2 bytes, 4 bytes)

### DISCUSSION THREAD

**Discussion by Ehsanwww** [ggml-org/whisper.cpp#1573] [General]
Hello @cjheath, thanks for great asset, in order to detect text from voice or a sound clip, that data that should be passed into  whisper_full() method, should be in what format? I mean mono/stereo and also 2 bytes or 4 bytes for each samples?

**Comment by Alumniminium** [comment]
16khz mono, 2 bytes (16bit) - also known as pcm_s16le

**Comment by cjheath** [comment]
> Hello @cjheath, thanks for great asset

I am not an author here. Please don't ping other users.
